lang/java/grpc/pom.xml

```diff
@@ -87,6 +92,12 @@
       <!-- use netty only for tests as user can use grpc transports other than Netty -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-codec-http2</artifactId>
+      <version>${netty-codec-http2.version}</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
The change does not seem to do anything with grpc. Did you patch this pom and the main pom by mistake?

Reply:
It does, apparently grpc has some implicit dependencies that both IntelliJ (2018.3) and Maven 3.5.4 aren't happy with in windows, so we need to set them explicit.